drivers/wdm/audio/legacy/wdmaud/sup.c

```diff
@@ -72,8 +64,8 @@ SetIrpIoStatus(
     Irp->IoStatus.Information = Length;
     Irp->IoStatus.Status = Status;
     IoCompleteRequest(Irp, IO_NO_INCREMENT);
-    return Status;
 
+    return Status;
```

Comment:
Does this stylish only change actually make sense? :thinking: You're simply adding a newline here.

Reply:
It aerates the code.

Reply:
What "aerating" does mean in this case?... :thinking:

Reply:
making the code less cluttered, or something.
But yeah, looking again at the function, in that case I don't think it was that necessary.

Reply:
> You're simply adding a newline here (before return)

Actually, I am swapping these 2 lines.